docs/reference/commandline/service_create.md

```diff
@@ -136,6 +136,130 @@ $ docker service create \
 For more information about labels, refer to [apply custom
 metadata](../../userguide/labels-custom-metadata.md).
 
+### Volumes and bind-mounts (--mount)
+
+Services use a more advanced notation to 
```

Comment:
incomplete sentence. recommend leaving out relative "advanced" qualifier for

The following table describes the options for defining mounts in a service: